widget/embedded/src/containers/QuoteInfo/QuoteInfo.tsx

```diff
@@ -55,25 +305,43 @@ export function QuoteInfo(props: PropTypes) {
 
   return showQuote ? (
     <QuoteContainer onClick={() => onClick?.(quote)}>
-      <Quote
-        quote={quote}
-        error={error}
-        showModalFee={showModalFee}
-        warning={warning}
-        tagHidden={tagHidden}
-        selected={selected}
-        type={type}
-        expanded={expanded}
-        onClickAllRoutes={onClickAllRoutes}
-        input={{
-          value: inputAmount,
-          usdValue: inputUsdValue?.toString() ?? '',
-        }}
-        output={{
-          value: outputAmount?.toString() ?? '',
-          usdValue: outputUsdValue?.toString() ?? '',
-        }}
-      />
+      {config?.variant === 'full-expanded' ? (
```

Comment:
![image](https://github.com/rango-exchange/rango-client/assets/123297941/d0741ede-451b-464c-8c3c-cb320d1b2082)
![image](https://github.com/rango-exchange/rango-client/assets/123297941/14d10aa7-2055-4df2-bd02-73ce52c08c35)
I think we have a type (list-item, swap-preview , ...) that should be considered in this condition.

Reply:
Fixed

Reply:
The confirm swap is fixed but
If you go to this url:
http://localhost:3002/routes?fromBlockchain=AVAX_CCHAIN&fromToken=USDT--0x9702230a8ea53601f5cd2dc00fdbc13d4df4a8c7&toBlockchain=BSC&toToken=USDC--0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d&fromAmount=5
you can see the Routes page still has an issue (in both loading & loaded state).
I don't know if we will disable the routes page in expanded-mode or not but with handling url manually the issue still exists.

Reply:
fixed